src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -164,30 +222,55 @@
      * file.
      */
     public static final String MSG_SUMMARY_JAVADOC = "summary.javaDoc";
+
     /**
      * A key is pointing to the warning message text in "messages.properties"
      * file.
      */
     public static final String MSG_SUMMARY_JAVADOC_MISSING = "summary.javaDoc.missing";
+
+    /**
+     * A key is pointing to the warning message text in "messages.properties" file.
+     */
+    public static final String MSG_SUMMARY_MISSING_PERIOD = "summary.javaDoc.missing.period";
+
     /**
      * This regexp is used to convert multiline javadoc to single line without stars.
      */
     private static final Pattern JAVADOC_MULTILINE_TO_SINGLELINE_PATTERN =
             Pattern.compile("\n[ ]+(\\*)|^[ ]+(\\*)");
 
+    /**
+     * This regexp is used to remove html tags, whitespaces and asterisks from string.
+     */
+    private static final Pattern REDUNDANT_ELEMENTS_PATTERN =
+            Pattern.compile("<[^>]*>|(, + *)|(\\*)");
+    /**
+     * This regexp is used to extract content inside summary javadoc tag from a string.
```

Comment:
What do you think about: `This regexp is used to extract the content of a summary javadoc tag.` ?

Reply:
Changed to your suggestion.

Reply:
Resolved.